WcaOnRails/spec/i18n_locale_files_spec.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require 'i18n-spec'
+
+Dir.glob(Rails.root.join('config', 'locales', '*.yml')).each do |locale_file|
+  RSpec.describe locale_file.to_s do
+    it { is_expected.to have_valid_pluralization_keys }
```

Comment:
Thanks for the link! I like the look of `it_behaves_like 'a valid locale file', locale_file`. Is is possible for us to use that, or does that check too much stuff?